src/menu-helpers.js

```diff
@@ -53,14 +54,16 @@ function unmerge(menu, item) {
   }
 }
 
-function findMatchingItemIndex(menu, { type, label, submenu }) {
+// function findMatchingItemIndex(menu, { type, label, submenu }) {
+function findMatchingItemIndex(menu, { type, id, submenu }) {
   if (type === 'separator') {
     return -1;
   }
   for (let index = 0; index < menu.length; index++) {
     const item = menu[index];
     if (
-      normalizeLabel(item.label) === normalizeLabel(label) &&
+      // normalizeLabel(item.label) === normalizeLabel(label) &&
```

Comment:
```suggestion
```

Reply:
I see. I delete the line, according to your advice.